src/components/data2.js

```diff
@@ -0,0 +1,25 @@
+export const questions = [
+  {
+    id: 1,
```

Comment:
ids might not have to be added here but can be rendered as you're iterating through the loop?